components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/internal/OAuthApplicationMgtListener.java

```diff
@@ -282,4 +294,79 @@ private void removeEntriesFromCache(ServiceProvider serviceProvider, String tena
         }
 
     }
+
+    /**
+     * Retrieves the value of SaaS property before application is updated
+     *
+     * @param serviceProvider ServiceProvider
+     * @param tenantDomain    String
+     * @throws IdentityApplicationManagementException
+     */
+    private void getSaaSPropertyValue(ServiceProvider serviceProvider, String tenantDomain) throws IdentityApplicationManagementException {
```

Comment:
It's not nice to have a method as "getX()" which doesn't return anything. So shall we rename this appropriately?